web/libs/editor/tests/e2e/tests/image.selected-region.test.js

```diff
@@ -44,40 +43,51 @@ async function testRegion(testType, toolAccelerator, I, LabelStudio, AtImageView
 
   I.amOnPage("/");
 
-  I.executeScript(initLabelStudio, params);
+  LabelStudio.init(params);
 
-  AtImageView.waitForImage();
+  LabelStudio.waitForObjectsReady();
   await AtImageView.lookForStage();
-  I.executeScript(waitForImage);
 
   I.say(`Select ${testType} & planet class`);
   I.pressKey(toolAccelerator);
   I.pressKey("1");
 
   I.say("There should be no regions initially");
-  AtSidebar.seeRegions(0);
+  AtOutliner.seeRegions(0);
 
   I.say(`${testType} initial region`);
-  await doDrawingAction(I, { msg: `Initial ${testType}`, fromX: 150, fromY: 110, toX: 150 + 50, toY: 110 + 50 });
+  await doDrawingAction(I, {
+    msg: `Initial ${testType}`,
+    fromX: 319 + 150,
```

Comment:
would be awesome to see these numbers as consts to understand the meaning behind them